src/App.js

```diff
@@ -2,6 +2,37 @@ import React from "react";
 import "./App.css";
 import { makeShuffledDeck } from "./utils.js";
 
+const maxRounds = 11;
+
+function TallyScores({ player1Score, player2Score, roundsLeft }) {
+  var para_top = (
+    <p>
+      Player 1 has won {player1Score} rounds. <br /> <br />
+      Player 2 has won {player2Score} rounds. <br /> <br />
+    </p>
+  );
+
+  var para_bottom = "";
+  if (roundsLeft === 0) {
+    if (player1Score > player2Score) {
+      para_bottom = <p> Player 1 wins this game!</p>;
+    } else if (player1Score < player2Score) {
+      para_bottom = <p>Player 2 wins this game!</p>;
+    } else {
+      para_bottom = <p>It's a tie!</p>;
+    }
+  } else {
+    para_bottom = <p>There are {roundsLeft} rounds left.</p>;
+  }
+
+  return (
+    <>
+      {para_top}
+      {para_bottom}
+    </>
+  );
```

Comment:
```suggestion

if (!roundsLeft) {
    return (
         <>
              {para_top}
              {player1Score > player2Score && <p> Player 1 wins this game!</p>}
              {player1Score < player2Score && <p>Player 2 wins this game!</p>}
              {player1Score === player2Score && <p>It's a tie!</p>}
         </>
    )
}

  return (
    <>
      {para_top}
     <p>There are {roundsLeft} rounds left.</p>
    </>
  );
```

I think you could utilize an early return here, to make the logic a bit more readable.